tests/functional/s3curl/tests.js

```diff
@@ -124,88 +122,155 @@ function provideRawOutput(args, cb) {
     });
 }
 
-describe('s3curl put and delete buckets', () => {
-    it('should put a valid bucket', done => {
-        provideRawOutput(['--createBucket', '--',
-            `${endpoint}/${bucket}`, '-v'],
-            httpCode => {
-                assert.strictEqual(httpCode, '200 OK');
-                done();
-            });
-    });
+function putObjects(filepath, objectPaths, cb) {
```

Comment:
this function can be use to delete objects and buckets, reducing a lot of tests :)